cmd/argocd-repo-server/commands/argocd_repo_server.go

```diff
@@ -128,6 +130,11 @@ func NewCommand() *cobra.Command {
 			helmRegistryMaxIndexSizeQuantity, err := resource.ParseQuantity(helmRegistryMaxIndexSize)
 			errors.CheckError(err)
 
+			var ociMediaTypesList []string
+			if ociLayerMediaTypes != "" {
+				ociMediaTypesList = strings.Split(ociLayerMediaTypes, ";")
```

Comment:
Usually I saw comma is being used instead of semicolon in such cases.
Is there a reason for choosing semicolon specifically?

Reply:
No specific reason, when @alexmt initially added `api-content-types` he used semicolon as a separator, and for this param I decided to go with the same convention.